packages/crew_mvdir.rb

```diff
@@ -3,28 +3,25 @@
 class Crew_mvdir < Package
   description 'Faster alternative to "rsync --remove-source-files dir1/ dir2/"'
   homepage 'https://github.com/chromebrew/crew-mvdir'
-  version '0.2-2' # Do not use @_ver here, it will break the installer.
+  version '0.3' # Do not use @_ver here, it will break the installer.
   license 'GPL-3'
   compatibility 'all'
-  source_url 'https://github.com/chromebrew/crew-mvdir.git'
-  git_hashtag '0.2'
-  binary_compression 'tar.zst'
-
-  binary_sha256({
-    aarch64: 'c6738e9eb4d6771c9349cbab24c5a0ff0802919e08931c979558bc0f2c33d719',
-     armv7l: 'c6738e9eb4d6771c9349cbab24c5a0ff0802919e08931c979558bc0f2c33d719',
-       i686: '94b576672248ce488bbc8c44f8ff83a5ef862264cd115ec5dea641d7ed2a740a',
-     x86_64: '7e8c79bded2e1c3d07c8d25c9773a9a8ecb23cd2e20169ff26ec6c09e9eeae41'
-  })
+  source_url 'https://github.com/supechicken/crew-mvdir.git'
+  git_hashtag 'e2b6b3fed81db907690a14f9953bb725b57b4e96'
 
   depends_on 'glibc' # R
 
   def self.build
-    system "mold -run cc #{CREW_COMMON_FLAGS} crew-mvdir.c -o crew-mvdir"
+    system *%w[mold -run ./build.sh]
   end
 
   def self.install
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
-    FileUtils.install 'crew-mvdir', "#{CREW_DEST_PREFIX}/bin/crew-mvdir", mode: 0o755
+    FileUtils.mkdir_p ["#{CREW_DEST_PREFIX}/bin", CREW_DEST_LIB_PREFIX, File.join(CREW_DEST_DIR, "#{CREW_LIB_PATH}/lib")]
+
+    Dir.chdir('builddir') do
+      FileUtils.install 'crew_mvdir.so', File.join(CREW_DEST_DIR, "#{CREW_LIB_PATH}/lib"), mode: 0o755
+      FileUtils.install 'crew-mvdir.so.1', "#{CREW_DEST_LIB_PREFIX}/crew-mvdir.so.1", mode: 0o755
```

Comment:
Why do we have one library file with a `-` and the other with a `_`?

Reply:
It is for distinguishing between the "real" shared libraries and the Ruby binding one (using `_` instead of `-` just like our `.rb` files)

Reply:
Are these supposed to be executable?